packages/preview-ui/README.md

```diff
@@ -1,5 +1,36 @@
 # Preview UI for Macaw
 
-**[Macaw](https://macaw.email/) is a simple library to streamline email templating.**
+The preview UI for Macaw helps view the compiled mjml file, markdown and JSON data. It is built with [react](https://reactjs.org/), and fetches the updated templates using [socket.io](https://socket.io/) from the macaw cli.
+ 
 
-Please check the [Macaw documentation](https://macaw.email/) for more information.
+![Macaw UI Preivew](./preview.png)
+
+**[Disclamer] This package is intended to be consumed internally by the macaw cli**
+
+## Getting Started
```

Comment:
Maybe something more like this to show this is regarding getting set up for development?

```suggestion
## Development setup
```